hangman.js

```diff
@@ -1 +1,26 @@
-console.log("welcome to hangman");
\ No newline at end of file
+//game variables
+var game = {
+  lettersInArray: [],
+  incorrectGuesses: [],
+  correctGuesses: [],
+  words: []
+}
+
+//Prompt Player One to Enter Secret Word
+function newWord() {
+  var new_word = prompt("Player 1: What is your secret word?");
```

Comment:
Just as a reminder, I would make sure to use camelCase (`var newWord`) instead  of snake case on line 11